.travis.yml

```diff
@@ -35,3 +35,8 @@ cache:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
 
+after_success:
+  - ./gradlew uploadGists
```

Comment:
Why not making it part of the test-downstream plugin? It would be easier to use by our customers!

Reply:
You mean not only applying it inside TestDownstreamPlugin but also making testDownstream be finalizedBy uploadGists? Shouldn't uploading logs be done in after_failure part of Travis build? It may take some time since these are network calls with potentially big attachments, so you would get later an information if build is successful.

Reply:
Let's avoid premature optimizations and focus on simplicity at first. If this indeed takes too long we can brainstorm how to make it better. There multiple options: do it in parallel, use after_failure/success.

Good question!